ww_commands.js

```diff
@@ -501,7 +501,7 @@ async function startSpel({ command, ack, say }) {
     const game = await queries.getActiveGame();
     const hiernamaals = await client.conversations.create({
       token: process.env.SLACK_BOT_TOKEN,
-      name: `${t('TEXTNAMEVIEWERCHANNEL')}_${game.gms_name.toLowerCase().split(' ').join('_')}`,
+      name: params[2].toLowerCase(),
```

Comment:
aah hier gebruik je de parameter, ik denk dat ik dit een vaste waarde zou laten en niet om een 2de kanaal naam zou vragen, deze naam is niet belangrijk voor het verhaal (de reden waarom je de naam van het hoofdkanaal moet meegeven :) )